docs/core/tools/dotnet-install-script.md

```diff
@@ -22,34 +22,31 @@ ms.assetid: b64e7e6f-ffb4-4fc8-b43b-5731c89479c2
 
 Windows:
```

Comment:
please update ms.date to reflect the topic has been recently reviewed

Reply:
done...